cumulusci/tasks/tests/test_pushfails.py

```diff
@@ -50,7 +50,7 @@ def setUp(self):
         self.task_config = TaskConfig({"options": {"request_id": "123"}})
 
     @mock.patch("cumulusci.tasks.push.pushfails.ReportPushFailures._update_credentials")
-    def test_run_task(self, *mocks):
+    def test_run_task(self, *_):
```

Comment:
pass `mock.Mock()` as a 2nd argument to `mock.patch` and then it won't try to pass it to the function.